resources/views/components/game/link-buttons/index.blade.php

```diff
@@ -104,7 +104,7 @@
         @can('viewAny', App\Models\Game::class)
             <x-game.link-buttons.game-link-button
                 icon="📜"
-                href="{{ '/setRequestors.php?g=' . $game->id }}"
+                href="route('game.requests.index', ['game' => $game])"
```

Comment:
I realize this isn't a new link, but I don't know where this button would appear. I don't see it on the desktop sidebar or the mobile info tab.